src/components/unit.tsx

```diff
@@ -34,21 +36,20 @@ const UnitStatus = (props: unitProps) => {
     <>
       {currentStatus !== STATUS_CODES.NOT_HOME && <>{status}</>}
       {currentStatus === STATUS_CODES.NOT_HOME && (
-        <Badge bg="secondary" text="light" className="me-1">
-          {status} {nhcount}
-        </Badge>
+        <span className="container-nothome">
+          <Image fluid src={envsvg} />
+          <Badge bg="none" className="badge-nothome" pill>
```

Comment:
<img width="132" alt="Screenshot 2022-11-21 at 10 27 17 PM" src="https://user-images.githubusercontent.com/40650158/203079707-fa2a8258-db10-45a6-b579-2c097894da86.png">

Looks good. I can see that you trying to blur the background image to make the nhcount more prominent. 

can I suggest that we just display the count as a simple div ? The Badge wrapper comes with some bloat styles which you are basically negating in your css. 
`<div className="nothome-count">{nhcount}</div>`

One way to achieve the "blur" effect is to apply opacity (maybe 0.5) to the image instead.
`<Image fluid src={nhsvg} className="class with opacity" />`